<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::SystemCommandWidget</class>
 <widget class="QWidget" name="Sekura::SystemCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="leAddress">
       <property name="text">
        <string>http://localhost:8081</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbCommands">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="te"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
